packages/core/logging/core-logging-server-internal/src/logging_system.ts

```diff
@@ -41,13 +42,17 @@ export class LoggingSystem implements ILoggingSystem {
   private readonly bufferAppender = new BufferAppender();
   private readonly loggers: Map<string, LoggerAdapter> = new Map();
   private readonly contextConfigs = new Map<string, LoggerContextConfigType>();
+  private readonly globalMeta: GlobalMeta = new Map();
 
   constructor() {}
 
   public get(...contextParts: string[]): Logger {
     const context = LoggingConfig.getLoggerContext(contextParts);
     if (!this.loggers.has(context)) {
-      this.loggers.set(context, new LoggerAdapter(this.createLogger(context, this.computedConfig)));
+      this.loggers.set(
+        context,
+        new LoggerAdapter(this.createLogger(context, this.computedConfig), this.globalMeta)
```

Comment:
Our `globalMeta` state needs to be set at the logging system level so that it can be applied to new loggers as they are created. We're basically storing the state here and then overwriting it in the `LoggerAdapter`, either via the constructor or `loggerAdapter.setGlobalMeta`.